src/features/recruitments/components/recruitment-form.tsx

```
@@ -0,0 +1,382 @@
+      techStacks: [],
+      openChatUrl: '',
+    },
+  });
+
+  const handleSubmit = form.handleSubmit((data) => {
+    const formData = new FormData();
+    formData.append('title', data.title);
+    if (data.summary) formData.append('summary', data.summary);
+    formData.append('content', data.content);
+    formData.append('projectType', data.projectType);
+    data.roles.forEach((role) => formData.append('roles', role));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)